widgets/pie/options.go

```diff
@@ -0,0 +1,50 @@
+package pie
```

Comment:
Would you mind editing the README.md as well? This is where we list all the available widgets, we could list this one too.

https://github.com/mum4k/termdash/blob/master/README.md